planning/mission_planner/launch/mission_planner.launch.py

```diff
@@ -43,12 +43,11 @@ def generate_launch_description():
             ("input/modified_goal", LaunchConfiguration("modified_goal_topic_name")),
             ("input/vector_map", LaunchConfiguration("map_topic_name")),
             ("debug/route_marker", LaunchConfiguration("visualization_topic_name")),
-            ("/localization/kinematic_state", LaunchConfiguration("odom_topic_name")),
         ],
         parameters=[
             LaunchConfiguration("mission_planner_param_path"),
             {"use_sim_time": LaunchConfiguration("use_sim_time")},
-            {"map_frame": LaunchConfiguration("map_frame_id")},
+            {"map_frame": "map"},
```

Comment:
Why not just leave this as a parameter?

Reply:
Because mission planning is only in map frame. Just to reduce the amouont of arguments.